### PR TITLE
made an input text component and modified button component

### PR DESCRIPTION
**Pull request by NancyAanchal**
made input text component
modified button component according to figma
added these components to dictionary search

### CODE REVIEW

File: nepalingo-web/src/components/Button.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {}
 const Button = forwardRef<HTMLButtonElement, ButtonProps>((props, ref) => {
   const { className, children, ...rest } = props;
 
-  const buttonClasses = `bg-blue-500 hover:md:bg-opacity-80 text-white font-bold py-2 px-4 rounded ${className}`;
+  const buttonClasses = `bg-[#D03641] hover:bg-opacity-80 text-white font-primary font-bold py-2 px-4 rounded-lg ${className}`;
```
Comment (SansCaar) [inline nepalingo-web/src/components/Button.tsx:8]: bg-primary  and remove the hover:bg-opacity-80 the cursor is enought

File: nepalingo-web/src/components/InputText.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+interface InputTextProps {
+  value: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  onKeyDown?: (e: React.KeyboardEvent<HTMLInputElement>) => void;
+  placeholder?: string;
+}
```
Comment (SansCaar) [inline nepalingo-web/src/components/InputText.tsx:8]: extends React.InputHTMLAttributes<HTMLInputElement>

File: nepalingo-web/src/components/InputText.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+const InputText: React.FC<InputTextProps> = ({
+  value,
```
Comment (SansCaar) [inline nepalingo-web/src/components/InputText.tsx:11]: add {
all the props,
...otherInputProps
}

File: nepalingo-web/src/components/InputText.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  placeholder,
+}) => {
+  return (
+    <input
+      type="text"
+      value={value}
+      onChange={onChange}
+      onKeyDown={onKeyDown}
+      placeholder={placeholder}
+      className="w-[785px] h-[60px] rounded-md p-[16px_12px] text-lg shadow-sm transition-colors duration-300 focus:border-blue-500 focus:shadow-lg bg-[#2B2B2B] text-white"
```
Comment (SansCaar) [inline nepalingo-web/src/components/InputText.tsx:23]: add a line
{...otherInputProps}

**Review by binamkayastha — APPROVED** [review]
noice

**Pull request merged**